docs/conf.py

```diff
@@ -6,65 +6,26 @@
 
 # -- Path setup --------------------------------------------------------------
 
-import datetime
 import os
 import sys
-
-# If extensions (or modules to document with autodoc) are in another directory,
-# add these directories to sys.path here. If the directory is relative to the
-# documentation root, use os.path.abspath to make it absolute, like shown:
-
-#sys.path.insert(0, os.path.abspath('../../'))
-#sys.path.insert(0, os.path.abspath('../'))
-#sys.path.insert(0, os.path.abspath('./'))
-#sys.path.insert(0, os.path.abspath('.'))
+import datetime
 
 # -- Project information -----------------------------------------------------
 
 project = 'Ubuntu Packaging Guide'
-copyright = f'Canonical Group Ltd, {datetime.date.today().year}'
+author = 'Canonical Group Ltd.'
+version = '2.0'
+copyright = f'{author}, {datetime.date.today().year}'
 
 # -- General configuration ---------------------------------------------------
 
-# If your documentation needs a minimal Sphinx version, state it here.
-needs_sphinx = '5.1.1'
-
-# Add any Sphinx extension module names here, as strings. They can be
-# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
-# ones.
-
+needs_sphinx = '4.3.2'
 extensions = [
-    'm2r2',
```

Comment:
I would prefer to keep m2r2 here if possible. It will give us flexibility to accommodate people writing in both markdown and reST if we can convert files submitted in md to rst.

Reply:
I am not keen of having 2 formats, instead we should encourage people to submit articles as markdown and I (and if I recall correctly Dave too) am happy to translate it to reST.

Reply:
m2r2 would do that automatically, but if you're both happy to do it by hand then there's no issue with removing the extension and this can be marked resolved

Reply:
The other issue with keeping `m2r2` is it's not packaged (in any release of Ubuntu); I can add a quilt patch to my packaging branch to excise the requirement when building the package but I quite liked that (currently) there are no patches at all in the packaging branch.

Still, while @dviererbe is absolutely correct that I'm happy to translate anything to reST, I'll defer to the wisdom of the documentation team here and remove the ... erm ... removal!